src/enums/nr-action.ts

```diff
@@ -2,12 +2,13 @@
 export enum NrAction {
   CANCEL = 'CANCEL',
   EDIT = 'EDIT',
-  REAPPLY = 'REAPPLY',
+  RENEW = 'REAPPLY', // REAPPLY in API is RENEW in UI
```

Comment:
NOTE !!!

We may need to sync this with the API, since the API may rename this (so that "reapply" is not confused with renew or resubmit).

Reply:
ATM I am assuming this action name is unchanged.

Reply:
Vysakh had a look at this and compared with the API, and the values are fine.

Vysakh was also gracious enough to test Renew and Upgrade with staff payment, and it's fine. (Retry also works and Resubmit is future.)